### PR TITLE
Feat : Quiz functionality added

### PR DESCRIPTION
**Pull request by rahul31agrawal**
Hi Folks,

I have added quiz functionality to my App
Please give your PR review

Thanks

### CODE REVIEW

**Review by sohan9819 — APPROVED** [review]
UI looks beautiful . Code also looks clean . Looks good to merge

File: src/pages/Quiz/Quiz.js [hunk 1]
```diff
@@ -1,8 +1,68 @@
...
+    <div className='quiz'>
+      <span className='subtitle'>Welcome, {name}</span>
+
+      {
+        questions ? (
+        <>
+        <div className="quizInfo">
+            <span>{questions[currQues].category}</span>
+            <span>
+              {/* {questions[currQues].difficulty} */}
```
Comment (surbhi84) [inline src/pages/Quiz/Quiz.js:42]: remove if not required, or you can add a comment why its here.

**Review by surbhi84 — APPROVED** [review]
- Code looks good.
- Functionalities are working.
- Liked the dataset for quizes.

File: src/pages/Quiz/Quiz.js (hunk 1 above)
Comment (rahul31agrawal) [reply src/pages/Quiz/Quiz.js:42]: ok

**Pull request merged**